src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not begin with an empty line. Please remove the empty line to adhere to best practices.